src/sites/all/themes/custom/creed/js/style.js

```diff
@@ -0,0 +1,29 @@
+
+// jQuery(document).ready(function(){
+// 	$(".row-1 col-1 img").hover(
+
+// 	function (event) {
+// 		var a = $('.row-1 col-1 img').attr('src');
+// 		$('#slideshow').src = a;
+// 	}
+// 	);
+// });
+
+(function ($) {
+    addImage= function() {
+        $('.slide_show .row-1 img').click(
+        		function(){
+        			var a = $(this).attr('src');
+        			$('#slideshow').attr('src', a)
+        		}
+        	);  
+    }
+
+}) (jQuery);
+
+(function ($) {
```

Comment:
@manjutapali @Ayush17251 try using drupal behaviours, which is the drupal way of attaching events.